AdobeStockImage/Api/AssetRepositoryInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Api;
+
+use Magento\AdobeStockImage\Api\Data\AssetInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+interface AssetRepositoryInterface
+{
+    /**
+     * Save asset
+     * @api
```

Comment:
I think we should mark the interface an API and ensure it contains only the API methods

Reply:
Hi @sivaschenko, I added this change. Can you check again please?

Reply:
Hi @sivaschenko, I see this change in my branch. Is that change that you are looking for? http://prntscr.com/nu7dmm